### PR TITLE
chore: ini introductory details

### PR DESCRIPTION
**Pull request by shunkakinoki**

### CODE REVIEW

File: apps/paper/pages/introduction/comparison.mdx [hunk 1]
```diff
@@ -1,10 +1,46 @@
...
+## Overview with other chain abstraction solutions
+
+|                        | [Light](https://light.so) | [NEAR](https://near.org) | [Particle Network](https://particle.network) |
+| ---------------------- | :------------------------ | :----------------------- | :------------------------------------------- |
+| Signature Aggregation  | ✅                        | 🛑                       | 🛑                                           |
+| EIP-4337 Support       | ✅                        | 🔴                       | ✅                                           |
+| Keystore Sync          | ✅                        | 🔶                       | 🔶                                           |
+| No Trust Assumption    | ✅                        | 🔶                       | 🔶                                           |
+| Programmatic Execution | ✅                        | 🛑                       | 🛑                                           |
+| Multi VMs              | 🛑                        | ✅                       | ✅                                           |
```
Comment (raymond2338) [inline apps/paper/pages/introduction/comparison.mdx:12]: Suggestion: Alt VMs cuz we do support multi VMs (it just that it has to be EVMs)

File: apps/paper/pages/introduction/philosophy.mdx [hunk 2]
```diff
@@ -8,6 +8,6 @@ Users should not have to deal with the burden of interacting with multiple chain
 
 User should have the freedom to express the atomicity of cross-chain interactions (optimistic, strict, conditional) rather than being enforced/limited by the rules of underlying network.
 
-## Modularity and Permissionless
+## Open Source and Permissionless
```
Comment (raymond2338) [inline apps/paper/pages/introduction/philosophy.mdx:11]: ngl but modluar sounds better lol. up to you

Comment (shunkakinoki) [reply apps/paper/pages/introduction/philosophy.mdx:11]: true! will fix haha

File: apps/paper/pages/introduction/comparison.mdx (hunk 1 above)
Comment (shunkakinoki) [reply apps/paper/pages/introduction/comparison.mdx:12]: Yes!! That's such a good point - thank you

**Pull request merged**